src/bin/stratis_dbusquery_version.rs

```diff
@@ -0,0 +1,30 @@
+use dbus::{
+    self,
+    arg::{RefArg, Variant},
+    blocking::Connection,
+};
+use lazy_static::lazy_static;
+use std::time::Duration;
+
+pub const DBUS_PROPS_IFACE: &str = "org.freedesktop.DBus.Properties";
```

Comment:
Please drop "pub" modifiers on all constants, since we won't and can't export them from here.

Reply:
Resolved.